src/layout/section/textFieldsSection/TextFieldsSection.tsx

```diff
@@ -38,16 +39,15 @@ export const TextFieldsSection = styled(TextFieldsSectionDefinition)`
   }
 
   & .kirk-textfields-section-content .kirk-textField {
-    padding: 0 ${space.s};
     flex-grow: 1;
     flex: 1;
   }
 
   & .kirk-textfields-section-content .kirk-textField:first-child {
-    padding: 0 ${space.s} 0 0;
+    padding-right: ${space.s}!important;
```

Comment:
nit: It seems browsers auto-correct that but I'd rather place a whitespace before `!important`

Reply:
nit: you can use && to increase specificity and avoid this **AWEFUL !important**